malcontent.yaml

```diff
@@ -38,6 +34,7 @@ test:
       packages:
         - openssl
         - crane
+        - yara
```

Comment:
Should this be yara-x here as well?

Reply:
See https://github.com/wolfi-dev/os/pull/39625/files#r1917190164 to avoid having this conversation in parallel.